helm/charts/determined/values.yaml

```diff
@@ -0,0 +1,42 @@
+# httpPort controls the port at which the listens for connections on.
+httpPort: 8080 
+
+# db sets the configurations for the database.
+db:
+  # To deploy your own postgres DB please provider a hostAddress.
+  # Note: if db hostAddress is provided Determined will skip deploying
+  # a postgres DB.
+  # hostAddress:
+
+  # Required db field.
```

Comment:
non-blocking: maybe clarify that these fields are required if and only if hostAddress isn't provided